src/VisualStudio/LiveShare/Impl/Shims/GoToDefinitionHandlerShim.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.LanguageServer.Handler;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Microsoft.VisualStudio.LiveShare.LanguageServices;
+
+namespace Microsoft.VisualStudio.LanguageServices.LiveShare.Shims
+{
+    internal class GoToDefinitionHandlerShim : AbstractLiveShareHandlerShim<TextDocumentPositionParams, object>
+    {
+        public GoToDefinitionHandlerShim(IEnumerable<Lazy<IRequestHandler, IRequestHandlerMetadata>> requestHandlers) : base(requestHandlers, Methods.TextDocumentDefinitionName)
+        {
+        }
+    }
+
+    [ExportLspRequestHandler(LiveShareConstants.CSharpContractName, Methods.TextDocumentDefinitionName)]
+    internal class CSharpFindImplementationsHandlerShim : GoToDefinitionHandlerShim
```

Comment:
1. i'm a bit confused.  how is 'FindImplementations' a 'GoToDefinition' feature?  
2. There is inconsistent spelling of `goto` in this PR.  Some places uses `Goto` some places uses `GoTo`.

Reply:
1.  Good catch, mis-named :)
2.  Will fix.